cuegui/cuegui/MenuActions.py

```diff
@@ -799,7 +806,7 @@ def getWhatThisDependsOn(self, rpcObjects=None):
     viewDepends_info = ["&View Dependencies...", None, "log"]
     def viewDepends(self, rpcObjects=None):
         frames = self._getOnlyFrameObjects(rpcObjects)
-        from DependDialog import DependDialog
+        from .DependDialog import DependDialog
```

Comment:
Turns out these imports were deferred to prevent problems arising from circular dependencies. So moving them required some work to clean up the imports. I ended up standardizing to absolute imports everywhere as the best option.